Consul/Client_GetRequests.cs

```diff
@@ -306,6 +352,27 @@ protected override void ApplyOptions(ConsulClientConfiguration clientConfig)
             {
                 Params["dc"] = Options.Datacenter;
             }
+
+            if (Options.UseCache && Options.Consistency != ConsistencyMode.Consistent)
```

Comment:
If user set invalid combination of options, he should get an error from consul. We shouldn't try to silently fix it for him. The reason is that we don't really know what was the intent of the user. Whether he wanted to use cache or to use consistent mode?
```suggestion
            if (Options.UseCache)
```

Reply:
Sorry about this. I meant to remove this one as well.